tools/packaging/kernel/configs/fragments/whitelist.conf

```diff
@@ -14,4 +14,4 @@ CONFIG_HAVE_NET_DSA
 CONFIG_NF_LOG_COMMON
 CONFIG_MANDATORY_FILE_LOCKING
 CONFIG_ARM64_UAO
-CONFIG_VFIO_MDEV_DEVICE
+CONFIG_VFIO_MDEV_DEVICE
```

Comment:
This change is not needed / shouldn't be part of the PR.

Reply:
Added back the newline. Done.